redash/query_runner/databricks.py

```diff
@@ -157,10 +174,18 @@ def run_query(self, query, user):
 
                 rows = [
                     dict(zip((column["name"] for column in columns), row))
-                    for row in data
+                    for row in result_set
                 ]
 
                 data = {"columns": columns, "rows": rows}
+
+                if (
+                    len(result_set) >= settings.DATABRICKS_ROW_LIMIT
```

Comment:
```suggestion
                    len(result_set) >= ROW_LIMIT
```

Reply:
Since `settings` no longer includes DATABRICKS_ROW_LIMIT you need to use the constant declared in `databricks.py`